src/Extensions/RestoreContent/RestoreContentEditingConfig.ts

```diff
@@ -5,7 +5,6 @@ extendMenu((menu) => {
 
   menu.insert({
     enabled: canWrite(),
-    icon: 'https://cdn.jsdelivr.net/npm/bootstrap-icons/icons/stars.svg',
```

Comment:
`bootstrap-icons` is now available as a package in `main`. So you could import `stars.svg` from there.